src/Core/Configurations/RuntimeConfigValidator.cs

```diff
@@ -267,6 +269,21 @@ public static void ValidateEntityConfiguration(RuntimeConfig runtimeConfig)
             }
         }
 
+        /// <summary>
+        /// Helper method to validate and let users know whether insignificant properties are present in the REST field.
+        /// Currently, it checks for the presence of Methods property when the entity type is table/view and logs a warning.
+        /// Methods property plays a role only in case of stored procedures.
+        /// </summary>
+        /// <param name="entity">Entity object for which validation is performed</param>
+        /// <param name="entityName">Name of the entity</param>
+        private void ValidatePresenceOfExtraRestProperties(Entity entity, string entityName)
+        {
+            if (entity.Source.Type != EntitySourceType.StoredProcedure && entity.Rest.Methods is not null && entity.Rest.Methods.Length > 0)
```

Comment:
Super nit: Either use `is not` or `!=` for both the comparisons. Looks better :)

Reply:
Since this method is called from only one place, do we want to create a method at all?

Reply:
My thought process behind introducing a separate method is to group all validations related to the presence of extra/insignificant properties in a single place - At the moment, only one field is being validated but more such validations can be added in the future.

Also, having a separate method makes it re-usable in another place in the future if needed